src/app/components/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,7 @@
+.img {
+  width: 10em;
+  height: 10em;
+  border-radius: 100%;
```

Comment:
I think if you are looking for a round shape just go to 50%

Reply:
Thanks! I changed it.